content/metadata.yaml

```diff
@@ -1,10 +1,11 @@
 ---
-title: "Mining Heterogenous Relationships from Pubmed Abstracts Using Weak Supervision"
+title: "Reusing Label Functions to Scale the Extraction of Relationships From Biomedical Abstracts to Multiple Types"
```

Comment:
```suggestion
title: "Reusing Label Functions to Extract Multiple Types of Biomedical Relationships From Biomedical Abstracts at Scale"
```

Reply:
see what you and @dhimmel think of this. The order on the other version made it hard to read for me. It sounded like the scaling was biomedical abstracts to multiple types.